### PR TITLE
Integration spec

### PR DESCRIPTION
**Pull request by Hassaanjbaig-code**
### In this pull request 

- Add Integration specs 
     - of user (home and show)
     - of post (home and show)

I use Capybara to write a test  

image

image

### CODE REVIEW

File: spec/views/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+
+RSpec.describe Post, type: :system do
+  describe 'index' do
```
Comment (tugsem) [inline spec/views/post_spec.rb:5]: - [ ] In the user post index page, kindly add these tests either as the requirements ask:
- Tests that asserts you can see a section for pagination if there are more posts than fit on the view.
- Tests that asserts you can see the number of posts the user has written.

File: spec/views/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+
+RSpec.describe User, type: :system do
+  describe 'index' do
+    before do
```
Comment (tugsem) [inline spec/views/users_spec.rb:6]: - [x] Please add this test to user index page as the requirements ask: 
- Tests that asserts you can see the profile picture for each user.

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @Hassaanjbaig-code  and @rbhatt1999 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of gitflow
✔️professional documentation

## Required Changes ♻️
- [ ] Kindly add a screenshot of the outcome of your tests running,
- [ ] Kindly include a screenshots of your logs before and after the fix.
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/views/post_spec.rb (hunk 1 above)
Comment (Hassaanjbaig-code) [reply spec/views/post_spec.rb:5]: @tugsem How are you 

Can you tell me how I can create this test 
 Tests that asserts you can see a section for pagination if there are more posts than fit on the view.

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥💯💯


## Highlights 🔥🔥🔥🔥
- Followed Gitflow ✅
- Great working on the functionality 🔥🔥
- All requested changes are done✅🤩




## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**